environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - cartopy
   - cf-units
   - cftime
+  - csv
```

Comment:
Anyway we can avoid using the csv package? Less dependencies is better

Reply:
I can try to avoid it in this version since we only have two variables, but if we want to avoid it in the general version for all variables, we may need to transfer the master_map.csv to an yml file?

Reply:
Yaml or json maybe? I tend to find Jason easier to process in Python but that's a personal choice.

Reply:
I too suggest either `yaml` or `ujson` - but `csv` is a builtin Python module just like `os` so no need to add it as a third party dependency :+1:

Reply:
Removed changes